src/Views/BucketPanel/EntryDetail.test.tsx

```diff
@@ -322,4 +344,40 @@ describe("EntryDetail", () => {
       );
     });
   });
+
+  describe("Record Deletion", () => {
```

Comment:
Didn't get what you are testing here.  You have to mount the component and test it.  Like you did it before.